core/field_angle.js

```diff
@@ -235,12 +235,6 @@ Blockly.FieldAngle.prototype.setText = function(text) {
     return;
   }
   this.updateGraph_();
-  // Insert degree symbol.
```

Comment:
One last question. Should this block of code stay? In trying to update 
https://github.com/LLK/scratch-blocks/issues/843 to explain why https://github.com/google/blockly/pull/1044 didn't fix it, I noticed we accepted part of that change (this delete), but not the addition of render_. This is fine as it is now if it works as well as before as long as we update 843.  What was the js error? textElement_ is undefined? Was it here or in render_?